### ISSUE TITLE
Bug: Help page transit agency link focus rings aren't rounded on Firefox

### ISSUE THREAD

**Issue by machikoyasuda** [cal-itp/benefits#2547]
Help page transit agency contact info button focus rings aren't rounded on Firefox. Should be rounded.

## To Reproduce

1. Go to https://dev-benefits.calitp.org/help
2. Tab into Help page and go all the way down to transit agency help links.
![Image](https://github.com/user-attachments/assets/f8571227-81c9-4351-9a9c-3eb919f52ea2)

## Expected behavior

Focus ring should be rounded like the rest.

## Additional context

Adding `border-radius: ...` doesn't seem to work! 

![Image](https://github.com/user-attachments/assets/e3478c7c-ee15-4f16-a7e0-0759a0893f2b)

Looks fine on Mac/Chrome

![Image](https://github.com/user-attachments/assets/e49f8cc5-e796-4911-ad3e-e5b338247fbd)

## Browser
- Mac/Firefox
- Mac/Chrome

**Comment by thekaveman** [comment]
I'm on Linux/Firefox, and it seems like it's only the last two in the agency links:

![Image](https://github.com/user-attachments/assets/947700a1-477b-4945-8de0-0adcd0ad804e)

![Image](https://github.com/user-attachments/assets/d3ba7683-b0c6-4134-a475-170c5546dd38)

![Image](https://github.com/user-attachments/assets/c3e3ae11-1206-46a5-8516-fce1a14890e5)

**Comment by machikoyasuda** [comment]
I want to work on this in a slush week, after the major #2541 stuff is done.
